jwk_test.go

```diff
@@ -221,6 +221,9 @@ func TestMarshalNonPointer(t *testing.T) {
 }
 
 func TestMarshalUnmarshalInvalid(t *testing.T) {
+	invalidCoord := make([]byte, curveSize(ecTestKey256.Curve)+1)
+	invalidCoord[0] = 1
```

Comment:
Yeah, I'm trying to generate a big integer that is too large for the curve. That's why I'm making a new byte array with `curveSize + 1`, and then setting the first byte to `1` (because otherwise it's just zero).